### PR TITLE
Add Main page

### LINKED ISSUE #10: Interested Person Profile
# Home page
As a user I want to be greeted by login, see available dogs and access the search page and menu.

## Acceptance Criteria:
- [ ] A page where dogs who are in the systems longer are shown
- [ ] A burger button for the menu
- [ ] A button to search

### PR DESCRIPTION
**Pull request by danifosy**
Last style only PR for now. Will resolve #10 
Something is going on here, I'm not sure what but when I look at the commits and I'm confused. Can't wait to merge that stuff.

### CODE REVIEW

File: src/app/components/BackButton/BackButton.tsx [hunk 1]
```diff
@@ -7,7 +7,11 @@ export default function BackButton(): JSX.Element {
     history.back();
   }
   return (
+mainPage
+    <button className={styles.icon} onClick={() => history.back()}>
+
```
Comment (marwinburesch) [inline src/app/components/BackButton/BackButton.tsx:12]: Woof, this is weird and incorrect :)

File: src/app/pages/LoginPage/LoginPage.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import Typography from '../../components/Typography/Typography';
+import { Link } from 'react-router-dom';
+import styles from './LoginPage.module.css';
+
+export default function LoginPage(): JSX.Element {
+  return (
+    <main className={styles.login}>
+      <h2>
+        <Typography size="l">Einloggen</Typography>
+      </h2>
```
Comment (marwinburesch) [inline src/app/pages/LoginPage/LoginPage.tsx:13]: 👀 so many elements? `h2`, `h3` ;)

File: src/app/pages/LoginPage/LoginPage.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      <Link to="" className={styles.button}>
+        <SubmitButton icon="signup">Einloggen</SubmitButton>
+      </Link>
+      <Link to="/signupselection" className={styles.signupLink}>
+        <p>
+          <Typography size="s">
+            Noch keinen Account? <br />
+            Hier anmelden!
+          </Typography>
+        </p>
```
Comment (marwinburesch) [inline src/app/pages/LoginPage/LoginPage.tsx:29]: I have seen this paragraphception before...

**Review by marwinburesch — COMMENTED** [review]
Please make sure to use your components like you designed them :)

File: src/app/pages/LoginPage/LoginPage.tsx (hunk 2 above)
Comment (danifosy) [reply src/app/pages/LoginPage/LoginPage.tsx:13]: I deleted them and now they are back, merge conflict party D: Time for the generalFixes5 branch.

**Pull request merged**